ci/test_container.sh

```diff
@@ -101,6 +101,10 @@ echo "#####################"
 echo "Run integration tests for NVTabular"
 /nvtabular/ci/test_integration.sh $container $devices --report 1
 
+# Test Transformers4Rec
+echo "Run integration tests for Merlin Models"
+/models/ci/test_integration.sh $container $devices
```

Comment:
Is WB set to stream metrics by default? if it is, can we invert it, so that it is set not to report by default and we have to turn on reporting.

Reply:
Yes @jperez999 . For integration tests W&B is set to stream metrics by default. What could be a reason for disabling that?